Modernize README (#286)
microsoft/vcpkg-docs: vcpkg/readme/vcpkg-README.md

large catalog of open-source libraries and enterprise-ready features designed to
facilitate your development process with support for any build and project
systems. vcpkg is a C++ tool at heart and is written in C++ with scripts in
CMake. It is designed from the ground up to address the unique pain points C/C++
developers experience.

This tool and ecosystem are constantly evolving, and we always appreciate
contributions! Learn how to start contributing with our [packaging
tutorial](../get_started/get-started-adding-to-registry.md) and [maintainer
guide](../contributing/maintainer-guide.md).

# Get started

First, follow the [quick start guide](../get_started/get-started.md).

If a library you need is not present in the vcpkg registry, you can [open an
issue on the GitHub repository][contributing:submit-issue] where the vcpkg team and
community can see it, and potentially add the port to vcpkg. Or you can
[contribute the package yourself](../get_started/get-started-adding-to-registry.md).

After you've gotten vcpkg installed and working, you may wish to add
[tab completion](../commands/integrate.md#vcpkg-autocompletion) to your shell.

> vcpkg has you covered, it doesn't matter what build system you use:

Reviewer: ```suggestion
Whether you're using CMake, MSBuild, or any other build system, vcpkg has you covered:
```

* [vcpkg with CMake](../get_started/get-started.md)
* [vcpkg with MSBuild](../get_started/get-started-msbuild.md)
* [vcpkg with other build systems](../users/buildsystems/manual-integration.md)

> Or what editor:

Reviewer: ```suggestion
You can also use any editor:
```

* [vcpkg with Visual Studio](../get_started/get-started-vs.md)
* [vcpkg with Visual Sudio Code](../get_started/get-started-vscode.md)
* [vcpkg with
  CLion](<https://www.jetbrains.com/help/clion/package-management.html>)

# Use vcpkg

> Create a [manifest for your project's dependencies](../consume/manifest-mode.md)

Reviewer: Maybe showcase the `vcpkg new --application`/ `vcpkg add port fmt`?

(`vcpkg.json`):

```json
{
  "name": "my-project",
  "version": "0.1.0",
  "dependencies": [
    "fmt"
  ]
}
```

Or [install packages throught the command line](../consume/classic-mode.md):

```Console
./vcpkg install fmt
```

Then use one of our available build system integrations for
[CMake](../concepts/build-system-integration.md#cmake-integration),
[MSBuild](../concepts/build-system-integration.md#msbuild-integration) or 
[other build
systems](../concepts/build-system-integration.md#manual-integration).

For a short description of all available commands, run `vcpkg help`.
Run `vcpkg help [topic]` for details on a specific topic.

# Contribute

vcpkg is an open source project, and is thus built with your contributions. Here are some ways you can contribute:

> ]

Reviewer: ```suggestion

```
